guides/v2.3/payments-integrations/cardinal/cardinal.md

```diff
@@ -0,0 +1,290 @@
+---
```

Comment:
I can't tell whether the 2.3 version is identical to the 2.2 version without downloading the files and running a diff, but spot checks look the same, except the 2.3 version doesn't have a blank line at the end. If they are the same, create a symlink between 2.2 and 2.3. 

I'm assuming the versions are the same, or are similar enough that the edits I made in 2.2 also apply to 2.3.

Reply:
Actually, it's a symlink

Reply:
Symlink created